packages/backend/src/database/entities/savedCharts.ts

```diff
@@ -10,11 +10,12 @@ type DbSavedChart = {
     space_id: number;
     name: string;
     created_at: Date;
+    description: string | undefined;
 };
 
 export type SavedChartTable = Knex.CompositeTableType<
     DbSavedChart,
-    Pick<DbSavedChart, 'name' | 'space_id'>,
+    Pick<DbSavedChart, 'name' | 'space_id' | 'description'>,
     Pick<DbSavedChart, 'name'>
```

Comment:
@nathaliabj apologies for not covering this when we walked through the backend:

The arguments here are <Get type, Create type, Update type>

Since we want to update the description it should be:

```
export type SavedChartTable = Knex.CompositeTableType<
     DbSavedChart,
     Pick<DbSavedChart, 'name' | 'space_id' | 'description'>,
     Pick<DbSavedChart, 'name' | 'description'>
>
```